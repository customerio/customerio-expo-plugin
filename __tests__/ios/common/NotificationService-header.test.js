const fs = require("fs-extra");
const path = require("path");

const testProjectPath = path.join(__dirname, "../../../test-app");
const iosPath = path.join(testProjectPath, "ios");
const notificationServiceHeaderPath = path.join(iosPath, "NotificationService/NotificationService.h");

test("Plugin creates expected NotificationService.h", async () => {
  const content = await fs.readFile(notificationServiceHeaderPath, "utf8");

  expect(content).toMatchSnapshot();
});
