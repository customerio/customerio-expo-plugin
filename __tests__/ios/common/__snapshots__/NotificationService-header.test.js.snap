// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plugin creates expected NotificationService.h 1`] = `
"#import <UserNotifications/UserNotifications.h>

@interface NotificationService : UNNotificationServiceExtension

@end
"
`;
